docs/source/transactions.md

```diff
@@ -706,35 +899,44 @@ Command to change Pool's configuration
 
 #### AUTH_RULE
 
-Transaction to change authentication rules. 
-Authentication rules are stored in the State as key - value dictionary. This coincides with the storage structure in (auth_map)[auth_rule.md]. 
-If config ledger does not have a transaction for a given key, then a default rule defined in code is used.
-Key - some action in the format `prefix--txn_type--field--old_value--new_value`
-- `prefix` (enum: `ADD` or `EDIT`)
-- `txn_type` (string) - The type of transaction to change rights to. (Example: "0", "1", ...)
-- `field` (string) Change the rights to edit(add) some values from field.
-- `old_value` (string; optional) Old value of field, which can be changed to a new_value.
-- `new_value` (string) New value that can be used to fill the field.
-Value is a set of constraints on the execution of this action. There are two types of constraints:
-- AuthConstraint with `constraint_id = ROLE_CONSTRAINT_ID` and format `{constraint_id, role, sig_count, need_to_be_owner, metadata}`;
-- AuthConstraintOr with `constraint_id = OR_CONSTRAINT_ID` and `auth_constraints` - list of constraints;
-- AuthConstraintAnd with `constraint_id = AND_CONSTRAINT_ID` and `auth_constraints` - list of constraints;
+A command to change authentication rules. 
+Internally authentication rules are stored as a key-value dictionary: `{action} -> {auth_constraint}`.
+
+The list of actions is static and can be found in [auth_rules.md](auth_rules.md).
+There is a default Auth Constraint for every action (defined in [auth_rules.md](auth_rules.md)). 
+
+The `AUTH_RULE` command allows to change the Auth Constraint.
+So, it's not possible to register new actions by this command. But it's possible to override authentication constraints (values) for a given action.
+
+There are two types of constraints:
+- ConstraintEntity contains `{constraint_id, role, sig_count, need_to_be_owner, metadata}`
+- ConstraintList with format `{constraint_id, auth_constraints}` contains list of constraints.
```

Comment:
I think it would be better to specify possible values of `contraint_id`. Also `ContraintEntity` and `ContraintList` identifiers look a bit artificial here (they are defined, but are not referenced anywhere), so probably something like
```
- constraint_id: ROLE - this is a role constraint, following fields are required...
- constraint_id: OR or AND - this is a list constraint, following fields are required...
```
might look better.